google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -73,7 +73,7 @@
   private static final String MANIFEST_ARTIFACT_ID_KEY = "artifactId";
   private static final String MANIFEST_VERSION_KEY = "Implementation-Version";
   private static final String ARTIFACT_ID = "google-cloud-core";
-  private static final String LIBRARY_NAME = "gcloud-java";
```

Comment:
Ok, that confirms what I thought. 

The gRPC-based APIs in Java never actually set the UA.

So for the indefinite future, we'll keep the UA for BQ, Storage, Datastore, and Translate, and only use `x-goog-api-client` for gRPC-based clients.